ros_topic:
  lidar_topic: "/velodyne_points"
  imu_topic: "/handsfree/imu"
  # pose_topic: "/aft_mapped_to_init"
  # pose_topic: /global_planning_node/robot_pose
  pose_topic: /aft_mapped_path # a_loam
  odom_topic: "/handsfree/odom"
  # map_topic: "/cloud_registered" # fast_lio
  map_topic: "/velodyne_cloud_registered" # a_loam
  # map_topic: "/full_cloud_projected" 

  path_topic: "/path"

map:
  resolution: 0.08
  map_x_l: -20.0
  map_x_u: 20.0
  map_y_l: -20.0
  map_y_u: 20.0

  queue_size: 20 
  queue2_size: 15

planning: 
  step_size: 0.5 # 步长大小
  h_surf_car: 0.5
  goal_biased: 0.15
  goal_threshold: 0.2
  sub_goal_threshold: 0.5
  inherit_threshold: 0.3

  w_fit_plane: 1.0
  w_flatness: 1.0
  w_slope: 0.05
  w_sparsity: 0.1

  ratio_max: 0.05
  ratio_min: 0.02
  conv_thre: 0.1152

  radius_fit_plane: 0.4

  max_vel: 0.05
  max_acc: 0.1

  # simulation 0.35 actual 0.45
  plane_bottom: -0.57 
  planning_horizon: 20.0
  planning_time_horizon: 0

  # plane_bottom: -0.48
  plane_bottom: -0.57
  planning_horizon: 20.0
  planning_time_horizon: 0

run_time_log: true
run_time_print: false
 



